elements/pfe-styles/src/pfe-type.scss

```diff
@@ -0,0 +1,65 @@
+@import "../../pfe-sass/pfe-sass";
+
+.pfe-headline-xxl {
+    @extend %pfe-headline-xxl;
+}
+
+.pfe-headline-xl {
+    @extend %pfe-headline-xl;
+}
+
+.pfe-generic h1,
+.pfe-headline-alpha {
+    @extend %pfe-headline-alpha;
+}
+
+.pfe-generic h2,
+.pfe-headline-beta {
+    @extend %pfe-headline-beta;
+}
+
+.pfe-generic h3,
+.pfe-headline-gamma {
+    @extend %pfe-headline-gamma;
+}
+
+.pfe-generic h4,
+.pfe-headline-delta {
+    @extend %pfe-headline-delta;
+}
+
+.pfe-generic h5,
+.pfe-headline-epsilon {
+    @extend %pfe-headline-epsilon;
+}
+
+.pfe-generic h6,
+.pfe-headline-zeta {
+    @extend %pfe-headline-zeta;
+}
+
+.pfe-text-lg {
+    @extend %pfe-text-lg;
+}
+.pfe-generic p,
+.pfe-text {
+    @extend %pfe-text;
+}
+
+.pfe-text-sm {
+    @extend %pfe-text-sm;
+}
+
+.pfe-text-xs {
+    @extend %pfe-text-xs;
+}  
+
+.pfe-generic {
+    > * {
```

Comment:
I'm not sure about including margin in this file.  This feels very heavy handed.

Reply:
Per our chat, I don't want to be heavy-handed either, but at least pfe-generic is opt-in, which can help solve the margins problem.

To summarize, the problem is that the .pfe-headline classes do not currently carry any margins with them -- maybe they should and then we won't need to use an asterisk. But currently if you do something like:
```html
<h5 class="pfe-headline-alpha">hello world</h5>
```
the margin-bottom is determined by the h5. Which means that your choice of semantic markup is influencing the visuals, and we want to avoid that if possible. 

Maybe we should do something like this instead?
```css
[class^="pfe-headline"] {
    margin-bottom: 20px; 
}
```

Reply:
```
[class^="pfe-headline"]:not(:last-child) {
    margin-bottom: 20px; // should this be rems?
}
```